Client/components/Login.jsx

```diff
@@ -0,0 +1,194 @@
+import React, {Component} from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import '../public/style.css';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
+import AppStore from '../stores/AppStore';
+import logo from '../public/images/logo.png';
+import profile from '../public/images/blank-profile-pic.png';
+
+class Login extends Component {
+  
+/**
+ * 
+ * Login event
+ * @param {object} event event object
+ * @memberof Login
+ */
+login(event){
+  // 
+  event.preventDefault();
+  const email = this.refs.loginEmail.value.trim();
+  const password = this.refs.loginPassword.value.trim();
+  
+  let user = {
+    email : email,
+    password: password,
+    isAuthenticated: false,
+    profilePic: ''
+  }
+
+  if(this.refs.loginEmail.value === ''){
+    AppActions.receiveErrors('Please insert email');
+  }else if (this.refs.loginPassword.value === '')
+  {
+    AppActions.receiveErrors('Please insert password');
+  } else{
+    AppActions.login(user); 
+  }
+}
+
+/**
+ * Toggles between login and register forms
+ * @return {void} return void
+ * @memberof Login
+ */
+handleToggle(){
+    $('form').slideToggle();
+    AppActions.receiveErrors('');
+    this.refs.email.value = '';
+    this.refs.username.value = '';
+    this.refs.password.value = '';
+    this.refs.loginEmail.value = '';
+    this.refs.loginPassword.value = '';
+  };
+
+/**
+ * Sign function
+ * @return {void} return void
+ * @param {object} event event object
+ * @memberof Login
+ */
+signup(event){
+  event.preventDefault();
+  let user = {
+    email : this.refs.email.value.trim(),
```

Comment:
Please use your component's state for this.
Attach a handle change function to each input element, in that function set state.

Your signup function shouldn't be dealing with element values directly.

It should be taking values from your state